docs/lib/datastore/fragments/android/sync/40-clear.md

```diff
@@ -31,6 +33,23 @@ Amplify.Hub.subscribe(HubChannel.AUTH,
 )
 ```
 
+</amplify-block>
+
+<amplify-block name="RxJava">
+
+```java
+// Listen for sign out events.
+final String signedOutEventName = AuthChannelEventName.SIGNED_OUT.toString();
+
+RxAmplify.Hub.on(HubChannel.AUTH)
```

Comment:
:eyes: here please as I made this up.

Reply:
Looks good other than the (unexpected) note below, that `clear()` isn't in the bindings atm.